### ISSUE TITLE
Blank PDA Window, FPS issues and hotkey/default keysettings switching on disconnect.

### ISSUE THREAD

**Issue by Gmno** [tgstation/tgstation#61537]
This is my first github issue and apologize in advance for my layout.

Issue 1: 
Recently I noticed a weird glitch where if my client disconnects it would swap to a different keyset, so if I was on vanilla default it'd bug out to hotkey and vise versa. This happens whether the disconnect occurs during a round or between rounds, the only fix I know of is closing my client before the server resets and reconnecting after it boots up. If you try to reconnect in game the bug continues into the next round. This also creates a FPS drop for me whenever this bug occurs.

Issue 2:
My pda has been a blank white page all day (9/17/2021), it didn't impact my rounds too much but everything I tried to fix it didn't work. Logged into two different TG servers and its the same problem. Uninstalled the version I was using, reinstalled the one from the website and it didn't fix the issue. Whenever I tried to interact with the window it starts lagging the game for me. Most windows seem to work in game, but I know the genetics computer gave me a weird bug the one time I peeked at it today. 

Tried to ask around to see if it was something on my end that was broken, but none worked and was told to come here.

**Comment by Mothblocks** [comment]
Please split your bug reports into separate issues so they can be handled independently.

**Comment by Gmno** [comment]
I think that both these issues are linked somehow because all this happened after that PR featuring the new setup UI was put in. Before that I had no problems. I expect more bugs will come that'll just make this more confusing to explain if I keep playing.

Should I remove one of my issues still?

**Comment by Mothblocks** [comment]
Yes, because even though both could be connected to prefs menu (I'm skeptical the second one is, sounds just like standard cache failure to me), it's very likely they need two separate fixes.

**Comment by Gmno** [comment]
Done, I'll poke around to see if I missed a folder with ss13 files somewhere but I thought that I wiped everything. Will make a new issue if it remains and take screenshots of the various windows I know are bugged.

**Comment by Mothblocks** [comment]
I've heard it before, it's just not super high on my priority list.
